app/brave_settings_strings.grdp

```diff
@@ -568,6 +568,23 @@
     Allowed to use third-party cookies for legacy Google Sign-In
   </message>
 
+  <!-- Localhost access setting -->
+  <message name="IDS_SETTINGS_SITE_SETTINGS_LOCALHOST_ACCESS" desc="Label for localhost site settings.">
+    Localhost Access
+  </message>
+  <message name="IDS_SETTINGS_SITE_SETTINGS_LOCALHOST_ACCESS_ASK" desc="Label for localhost access site settings.">
+    Sites can request access to localhost resources
```

Comment:
I am not sure this is descriptive enough for an average user to understand. What exactly are localhost resources?

Reply:
Started a DM along with @rmcfadden3

Reply:
one note for folks to consider is that other parts of the platform call this "private network access".  I dont think that requires changing the name here, but something to consider if people are consdiering different names